modules/project-factory/factory-projects-object.tf

```diff
@@ -95,107 +96,111 @@ locals {
       )
       parent                     = null
       prefix                     = null
-      service_encryption_key_ids = {}
+      service_encryption_key_ids = null
       storage_location           = null
-      tag_bindings               = {}
-      services                   = []
-      service_accounts           = {}
-      vpc_sc = merge({
-        perimeter_name    = null
-        perimeter_bridges = []
-        is_dry_run        = false
-        }, try(local._projects_config.data_overrides.vpc_sc, {
-          perimeter_name    = null
-          perimeter_bridges = []
-          is_dry_run        = false
-        })
-      )
-      logging_data_access = {}
+      tag_bindings               = null
+      services                   = null
+      service_accounts           = null
+      vpc_sc                     = try(local._projects_config.data_overrides.vpc_sc, null)
```

Comment:
you sure we don't need to prime the optional parts of vpc sc?

Reply:
If this is null, it will be primed below

Reply:
I don't mean null, I mean an incomplete object.